src/main/java/io/github/externschool/planner/entity/User.java

```diff
@@ -27,8 +27,17 @@
     @Column(name = "encrypted_password")
     private String encryptedPassword;
 
-    public User(){
+    @ManyToMany(cascade = {
+            CascadeType.PERSIST,
+            CascadeType.MERGE
+    })
+    @JoinTable(name = "user_authority",
+            joinColumns = @JoinColumn(name = "user_id"),
+            inverseJoinColumns = @JoinColumn(name = "authority_id")
+    )
+    private List<Authority> authorities = new ArrayList<>();
```

Comment:
this field is not final to be initialized when declared

Reply:
It's best practice 

you can read more here https://stackoverflow.com/questions/20715143/to-initialize-or-not-initialize-jpa-relationship-mappings

But for many to many should use Set instead List 
you can read more here
https://www.thoughts-on-java.org/association-mappings-bag-list-set/

Reply:
thanks! i've never paid attention to that since JPA should always return a non-null empty list even if no results are found